### PR TITLE
A small update

### PR DESCRIPTION
**Pull request by julien-leclercq**
Hello, 
As I said in #1, here is my updated version of it. It was actually a nice getting started thing because I had to understand the API changes. 
I updated the dependencies to be able to build. 
I added a few module to match the `revery-quick-start` repo

### CODE REVIEW

**Comment by julien-leclercq** [comment]
It seems that the build is not passing but I have no idea how to fix this 🤷‍♂️

**Review by glennsl — COMMENTED** [review]
Awesome. Thank you!

The CI failures are due to outdated and broken CI configuration. I'm trying to fix that in #3, and will re-run them here once that has been merged.

Otherwise I've quickly looked through the code, and it looks pretty good to me. The formatting is pretty off though. I'm not sure if the CI checks OCaml formatting, but it might be an idea to run `esy format`. If it changes anything it does and will fail CI unless you check it in.

File: package.json [hunk 1]
```diff
@@ -8,23 +8,18 @@
     "buildsInSource": "_build"
   },
   "dependencies": {
-    "ocaml": "~4.7.0",
-    "revery": "^0.9.0",
-    "@opam/dune": "*"
+    "@esy-ocaml/reason": "^3.6.0",
+    "revery": "revery-ui/revery#431e515a00ee46f7bc614a10c2a219aae13179e0",
+    "ocaml": "4.10.0000"
   },
```
Comment (glennsl) [inline package.json]: `revery-packager` requires `esy-macdylibbundler` on macOS

```suggestion
    "@esy-ocaml/reason": "^3.6.0",
    "revery": "revery-ui/revery#431e515a00ee46f7bc614a10c2a219aae13179e0",
    "ocaml": "4.10.0000",
    "esy-macdylibbundler": "*"
  },
```

File: src/AnimatedText.ml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          |> Animation.ease Easing.ease
+          |> Animation.tween 50. 0.) in
+              (
+                Opacity.createElement
+                  ~opacity:animatedOpacity
+                  ~children: [
+                (Padding.createElement
+                  ~padding:6
+                  ~children:[
+                    (Text.createElement ~style:(Styles.text yOffset) ~text:text ~children:[] () [@JSX ])
```
Comment (glennsl) [inline src/AnimatedText.ml]: Missing label yields a warning

```suggestion
                    (Text.createElement ~style:(Styles.text ~yOffset) ~text:text ~children:[] () [@JSX ])
```

**Comment by glennsl** [comment]
Yay! It passes! :smile:

**Pull request merged**